src/main/java/net/minecraftforge/common/config/FieldWrapper.java

```diff
@@ -86,7 +86,8 @@ private MapWrapper(String category, Field field, Object instance)
             }
             catch (Exception e)
```

Comment:
Can you look into cases of `catch (Exception e)` like this where we immediately rethrow without logging, to see if we can limit what they catch?
This one can probably be `catch (IllegalAccessException e)`.
I think if we can avoid catching and rethrowing some things it will be an improvement, and you may be able to avoid using `Throwables` depending on what we actually catch.